packages/guides/src/RenderContext.php

```diff
@@ -81,6 +81,28 @@ public static function forDocument(
         return $self;
     }
 
+    public static function forSitemap(
```

Comment:
This method is not used anymore? So we can remove it?

Reply:
removed it